src/pkg/src/lib.rs

```diff
@@ -389,12 +389,7 @@ impl Pkg {
                     status(status_name.as_ptr(), size, 0, ud);
 
                     // Open destination file.
-                    let mut dest = std::fs::OpenOptions::new();
-
-                    dest.create_new(true);
-                    dest.write(true);
-
-                    let mut dest = match dest.open(&output) {
+                    let mut dest = match OpenOptions::new().write(true).create(true).open(&output) {
```

Comment:
This one need to be `create_new`, not `create` becuase we want to catch if the game has duplicated file.